### DISCUSSION TITLE
How do I slice yolo .txt files?

### DISCUSSION THREAD

**Discussion by richard-schwab** [obss/sahi#840] [General]
I was able to slice a yolo image using the documentation here: https://github.com/obss/sahi/blob/main/docs/slicing.md  

However, how do I slice the yolo .txt file that is part of it so I can train? Or is that not the workflow you recommend?

I was of the mind that I should slice up my database and train it in yolov8 and then use this to predict. Is that not correct? There isn't a training.md in the docs folder to give recommendations.

**Comment by fcakyon** [comment]
sahi supports only slicing coco annotations. 

you can 
1. slice coco annotations, 
2. convert sliced coco files into yolo format using sahi
3. train yolov5/yolov8 with the converted yolo txt files

**Reply by richard-schwab** [reply]
But why, if yolo trains on yolo annotations?
